src/conversion.jl

```diff
@@ -15,7 +15,7 @@ function convert{R, C, R2, C2, T}(MT::Type{Mat{R, C, T}}, b::Mat{R2, C2, T})
 end
 
 #conversion
-convert{T <: Tuple}(::Type{T}, x::Real)                   = ntuple(ConstFunctor(eltype(T)(x)), Val{length(T.parameters),})
+convert{N}(T::Type{NTuple{N}}, x::Real) = ntuple(ConstFunctor(eltype(T)(x)), Val{N})
```

Comment:
Not to say that the change here is bad, but the very existence of this `convert()` definition outside `Base` seems a little dodgy.  Do we really need it?